koku/api/report/test/test_queries.py

```diff
@@ -422,14 +430,50 @@ def test_percent_delta_zero_division(self):
         pd = rqh._percent_delta(10, 0)
         self.assertEqual(pd, None)
 
+    def test_get_search_filter_with_exclude(self):
```

Comment:
Could we add a couple more test cases?

- One with a list of the same resource excluded ... like 2 or more projects
- One with multiple dimensions of exclude like gcp_project and service etc. (So we can see this functioning in multiple cases)

Reply:
I have more complete testing suite written in this PR: https://github.com/project-koku/koku/pull/3890/files

I have been debating merging it into this one or keeping it separate; however, I think it may make it easier to review this pull request so I am gonna merge it in.